docs/.vuepress/components/HelloWorld.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <div>
     <tabs :options="{ useUrlFragment: false }" @changed="tabClicked">
-      <tab v-for="item in items" :name="item.name" :id="item.tabId">
-        <CodeSandboxIframe :id="item.codeSandboxId" :title="item.title" v-if="isSelected(item.tabId)">
+      <tab v-for="demo in this.demos" :name="demo.name" :id="getTabId(demo.name)">
+        <CodeSandboxIframe
+          :id="demo.codeSandboxId"
+          :title="demo.title"
+          :selectedFile="demo.selectedFile"
+          v-if="isSelected(getTabId(demo.name))">
         </CodeSandboxIframe>
       </tab>
       <tab name="Vue" id="vue" :is-disabled="true"></tab>
```

Comment:
Could I please to pass disabled tabs as a props too?

Reply:
As we discussed, it will be completely removed soon.

Reply:
ok, as these tabs will be replaced by working demos in few days (#8265 #8404 ) this comment isn't worth caring about.